### PR TITLE
add references to template repository

### PR DESCRIPTION
**Pull request by MattWellie**
added a reference to the template when creating a new repo
small changes for the linter
didn't alter the rest of the document structure

### CODE REVIEW

**Comment by vladsaveliev** [comment]
Awesome! Would be also nice to also replace the instructions here with references to to the template: https://github.com/populationgenomics/team-docs/blob/main/new_repository.md#setting-up-pre-commit

**Comment by MattWellie** [comment]
@vladsaveliev  I've altered the content a little, moving the content included in the template into a separate document, and referencing it. I'm not sure if that's a good way to arrange it, but it means the setup instructions are still present if someone wants to set up a non-template repo

File: pre-commit.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Setting up Pre-Commit
+
+When creating a new empty repository, please add these configuration files into the repository
```
Comment (lgruen) [inline pre-commit.md:3]: Should we point out here that for a Python-based repository, using the template is the preferred approach?

**Review by lgruen — APPROVED** [review]

**Comment by MattWellie** [comment]
Waiting to rebase on https://github.com/populationgenomics/team-docs/pull/116, which will correct the embedded links

**Comment by MattWellie** [comment]
rebased on top of recent changes

**Review by vladsaveliev — APPROVED** [review]

**Comment by lgruen** [comment]
Is this ready to be merged?

**Comment by MattWellie** [comment]
I'll double check locally, I thought the rebase would fix those linting fails

**Comment by MattWellie** [comment]
ironically the pre-commit.md file was causing linter issues

**Pull request merged**